docs/learn-spree.md

```diff
@@ -4,11 +4,11 @@ title: SPREE
 sidebar_label: SPREE
 ---
 
-SPREE (Shared Protected Runtime Execution Enclaves), also known as "trust wormholes," allows parachains to trust one another, regardless of how they upgrade and evolve.
+SPREE (Shared Protected Runtime Execution Enclaves), also known as "trust wormholes," allow parachains to trust one another, regardless of how they upgrade and evolve.
```

Comment:
We definitely could use more info on SPREE, but we don't have it all yet and I think it would also be under a separate PR instead of this generic fix-up.  Filed a new issue, https://github.com/w3f/polkadot-wiki/issues/64

Reply:
@Swader Please resolve if you're OK with that